service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/CaseSubmissionController.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.hmcts.reform.fpl.controllers;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.fpl.templates.DocumentTemplates;
+import uk.gov.hmcts.reform.pdf.generator.HTMLToPDFConverter;
+
+import java.util.Map;
+import javax.validation.constraints.NotNull;
+
+
+@RestController
+@RequestMapping("/callback/case-submission")
+public class CaseSubmissionController {
+
+    private final HTMLToPDFConverter converter = new HTMLToPDFConverter();
+    private final DocumentTemplates documentTemplates = new DocumentTemplates();
```

Comment:
Please inject instance of DocumentTemplates using @Autowire

Reply:
Please remove ` = new DocumentTemplates()` part as this is no longer needed if DI (auto wiring) is used.